src/JestConfigEditor.ts

```diff
@@ -0,0 +1,35 @@
+import { Config, ConfigEditor } from 'stryker-api/config';
+import DefaultConfigLoader from './configLoaders/DefaultConfigLoader';
+import ConfigLoader from './configLoaders/ConfigLoader';
+import * as fs from 'fs';
+import ReactConfigLoader from './configLoaders/ReactConfigLoader';
+
+export default class JestConfigEditor implements ConfigEditor {
+  public edit(config: Config): void {
+    let project = 'default';
+
+    if (config.jest && config.jest.project) {
+      project = config.jest.project;
+    }
+
+    config.jest = config.jest || {};
```

Comment:
If you move this line to the start of the method, you can remove the if-statement above and simply write `let project = config.jest.project || 'default';` if I'm correct.

Reply:
Well if the jest property does not exist it will return with an error saying that it could not find `project` on `undefined`.

Reply:
This would work though:

```javascript
  public edit(strykerConfig: Config): void {
    strykerConfig.jest = strykerConfig.jest || {};
    strykerConfig.jest.project = strykerConfig.jest.project || 'default';
    strykerConfig.jest.config = strykerConfig.jest.config || this.getConfigLoader(strykerConfig.jest.project).loadConfig();
  }
```